datamodel/openapi/openapi-api-sample/src/test/java/com/sap/cloud/sdk/datamodel/openapi/sample/api/DeserializationTest.java

```diff
@@ -37,6 +37,7 @@ void testFullResponse()
               "name": "Cola",
               "brand": "Coca-Cola",
               "quantity": 100,
+              "packaging" : null,
```

Comment:
shouldn't this be something like:
```suggestion
              "packaging" : "new-value",
```

In particular, shouldn't this be non-null, because the spec explicitly declares it nonnull?